app/org/sagebionetworks/bridge/services/SchedulePlanService.java

```diff
@@ -120,6 +121,13 @@ public void deleteSchedulePlan(StudyIdentifier studyIdentifier, String guid) {
         schedulePlanDao.deleteSchedulePlan(studyIdentifier, guid);
     }
     
+    public void deleteSchedulePlanPermanently(StudyIdentifier studyIdentifier, String guid) {
+        checkNotNull(studyIdentifier);
+        checkNotNull(isNotBlank(guid));
+        
+        schedulePlanDao.deleteSchedulePlanPermanently(studyIdentifier, guid);
+    }
```

Comment:
Unit test for delete permanently?

Reply:
Added to mock test (neither delete call had a separate test SFAICT).